### PR TITLE
Add location capabilities

### PR DESCRIPTION
**Pull request by jpjoyal**
image

### CODE REVIEW

File: components/ChannelCapabilitiesGrid/index.js [hunk 1]
```diff
@@ -24,11 +24,11 @@ export default class ChannelCapabilitiesGrid extends Component {
                    <SectionHeader>
                        Content types
                    </SectionHeader>
-                   { this.generateRows(['text', 'image', 'gif', 'emoji']) }
+                   { this.generateRows(['text', 'image', 'location', 'gif', 'emoji']) }
                    <SectionHeader>
                        Action types
                    </SectionHeader>
-                   { this.generateRows(['link', 'buy', 'postback', 'replies']) }
+                   { this.generateRows(['link', 'buy', 'postback', 'replies','locationRequest']) }
```
Comment (dannytranlx) [inline components/ChannelCapabilitiesGrid/index.js]: Man, feels bad for whitespace after commas.

File: data/channelCapabilities.js [hunk 2]
```diff
@@ -81,6 +81,15 @@ export const CAPABILITIES = {
         link: '/guide/web-messenger#alternate-channels',
         send: false,
         receive: false
+    },
+    location : {
+        name: 'Location',
+        send: true,
+        receive: true
+    },
+    locationRequest :{
```
Comment (dannytranlx) [inline data/channelCapabilities.js]: Those colons

File: data/channelCapabilities.js [hunk 3]
```diff
@@ -181,7 +197,14 @@ const channelData = {
                 send: SUPPORT_LEVEL.NONE,
                 receive: SUPPORT_LEVEL.FULL
             },
-            webMessengerLinking: SUPPORT_LEVEL.FULL
+            webMessengerLinking: SUPPORT_LEVEL.FULL,
+            location:{
```
Comment (lemieux) [inline data/channelCapabilities.js]: missing space... looks like it needs a bit formatter help.

**Comment by jpjoyal** [comment]
Thanks @dannytranlx I'll merge now. This can go out event if android and web are not completely out.

**Pull request merged**